Bindings/bindings.json

```diff
@@ -1474,6 +1474,14 @@
                     ]
                 },
                 {
+                    "name": "apiKey",
+                    "value": "string",
+                    "defaultValue": "SendGridApiKey",
```

Comment:
Ok, I can see value in keeping the better name you have anyways. We'll just have to include details in the release notes.